experimenter/experimenter/experiments/models.py

```diff
@@ -726,10 +726,56 @@ def should_show_remote_settings_pending(self, reviewer):
             self.PublishStatus.WAITING,
         ) and self.can_review(reviewer)
 
+    def review_messages(self):
+        if self.status_next == self.Status.COMPLETE:
+            return NimbusUIConstants.REVIEW_REQUEST_MESSAGES["END_EXPERIMENT"]
+        elif self.is_paused:
+            return NimbusUIConstants.REVIEW_REQUEST_MESSAGES["END_ENROLLMENT"]
+        else:
+            return NimbusUIConstants.REVIEW_REQUEST_MESSAGES["LAUNCH_EXPERIMENT"]
+
+    @property
+    def remote_settings_pending_message(self):
+        if self.publish_status in (
+            self.PublishStatus.APPROVED,
+            self.PublishStatus.WAITING,
+        ):
+            return self.review_messages()
+
+    @property
+    def review_request_action_type(self):
+        return self.review_messages()
+
     @property
     def should_show_timeout_message(self):
         return self.changes.latest_timeout()
 
+    @property
+    def should_show_end_enrollment(self):
+        return self.is_enrollment
```

Comment:
`is_enrollment` isn't technically grammatically correct, it would be `is_enrolling`